src/assets/scripts/client/parsers/scopeCommandParser/ScopeCommandModel.js

```diff
@@ -0,0 +1,66 @@
+import _first from 'lodash/first';
+import _has from 'lodash/has';
+import _last from 'lodash/last';
+import _without from 'lodash/without';
+import _values from 'lodash/values';
+import {
+    IMPLIED_COMMANDS,
+    EXPLICIT_COMMANDS,
+    COMMAND_FUNCTIONS
+} from './scopeCommandMap';
+
+// FIXME: Remove dummy sector codes
+const SECTOR_HANDOFF_CODES = ['18', '19', '10', '12'];
+
+/**
+ * A command to be interpreted by the scope itself
+ * Used for manipulation of the display and other ATC tasks
+ *
+ * @class ScopeCommand
+ */
+export default class ScopeCommand {
+    constructor(command) {
+        this.aircraftReference = '';
+        this.commandArguments = [];
+        this.commandFunction = '';
+
+        this._init(command);
+    }
+
+    _init(command) {
+        const pieces = command.toUpperCase().split(' ');
+
+        this.aircraftReference = _last(pieces);
+        this.commandFunction = this._extractCommandFunction(pieces);
+        this.commandArguments = this._extractCommandArguments(pieces);
+    }
+
+    _extractCommandArguments(command) {
+        if (_values(IMPLIED_COMMANDS).indexOf(this.commandFunction) !== -1) {
+            return _without(command, this.aircraftReference);
+        }
+
+        return command.splice(1, command.length - 2);
+    }
+
+    _extractCommandFunction(command) {
+        const firstElement = _first(command);
+        if (_has(EXPLICIT_COMMANDS, firstElement)) {
+            return EXPLICIT_COMMANDS[firstElement];
+        }
+
+        if (SECTOR_HANDOFF_CODES.indexOf(firstElement) !== -1) {
+            return COMMAND_FUNCTIONS.INITIATE_HANDOFF;
+        }
+
+        if (firstElement.indexOf('/') !== -1 || firstElement.length < 2) {
```

Comment:
what does the `/` mean?  what is it for?

Reply:
The only implied command a `/` is used with is the move data block command. I tried for a while to figure out how or where to neatly extract it to, but all options were clunky so I just left it there.